.github/workflows/build-and-push-docker-image.yml

```diff
@@ -1,32 +1,32 @@
-name: 'Build and push Docker image'
+name: Build and push image to ECR
 on:
   push:
     branches:
-      - 'dev-cere'
+      - cere-master
+      
 env:
-  ECR_REPOSITORY: 'crb-substrate-testnet-node'
-  IMAGE_TAG: ${{ github.run_number }}-${{ github.sha }}
+  ECR_REPOSITORY: crb-substrate-testnet-node
```

Comment:
We need to create a repo `pos-network-node` in private registry. Please, work with #devops team to do it.

Reply:
Requested devops team https://www.notion.so/cere/Create-new-private-ECR-repository-5b3e95b9023a4f3d89459382e5cc62dc